### PR TITLE
feat: bump yaru and flutter versions

### PR DESCRIPTION
**Pull request by d-loose**
See https://github.com/canonical/ubuntu-desktop-provision/pull/179

### CODE REVIEW

**Comment by Feichtmeier** [comment]
Note: if you want the gtk4 paned look, you just need to make the title bars transparent and set the border to borderside.none

File: pubspec.yaml [hunk 1]
```diff
@@ -4,10 +4,8 @@ version: 1.0.0
 publish_to: 'none'
 
 environment:
-  # fix Flutter version to 3.10.6
-  # (exact version bug: https://github.com/dart-lang/sdk/issues/52672)
-  sdk: '>=3.0.6 <3.0.7'
-  flutter: '>=3.10.6 <3.10.7'
+  sdk: '>=3.1.0 <4.0.0'
+  flutter: '>=3.13.0'
```
Comment (tim-hm) [inline pubspec.yaml:8]: Same comment as in firmware-updater about fixing to a specific version rather than a range

**Review by tim-hm — APPROVED** [review]
One comment ref versions but otherwise lgtm

**Comment by tim-hm** [comment]
> Note: if you want the gtk4 paned look, you just need to make the title bars transparent and set the border to borderside.none

We are planning on updating the visuals ... just not queit yet as we have a lot of plubing to address first.

**Comment by Feichtmeier** [comment]
> .

Sure, just wanted to give a heads up that some things are auto changing with the new version, for example the sidebar has darker/lighter color now in light/dark themes (noted and acked by elio, too)
Other changes are opt-in if wanted

File: pubspec.yaml (hunk 1 above)
Comment (d-loose) [reply pubspec.yaml]: Fixed it to 3.13.7 here as well

**Pull request merged**